vars/uploadMinSnapshotsToS3.groovy

```diff
@@ -19,6 +19,10 @@ void call(Map args = [:]) {
 
     def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: manifest))
     String productName = inputManifest.build.getFilename()
+    String version_plain = inputManifest.build.version
+    String qualifier = inputManifest.build.qualifier ? '-' + inputManifest.build.qualifier : ''
+    String revision = version_plain + qualifier + '-SNAPSHOT'
```

Comment:
We build with snapshot=true. https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/publish-min-snapshots.jenkinsfile#L73
Do we still need revision here?

Reply:
We do, revision is more for the `qualifier`, such as `3.0.0-beta1`, `3.0.0-rc1`.